examples/contracts/amm_adaptive.py

```diff
@@ -0,0 +1,180 @@
+import json
```

Comment:
@mpaya5 nice to add this one, but let's do it on its own PR please and also with the new GenVM syntax

Reply:
I deleted the amm_adaptive.py